api/app/controllers/spree/api/checkouts_controller.rb

```diff
@@ -84,6 +77,33 @@ def user_id
           params[:order][:user_id] if params[:order]
         end
 
+        def update_params
+          if update_params = massaged_params[:order]
```

Comment:
Seems a bit bike-shedding-ish...

Also, in this case I like how the current form calls out the fact that we may want to discontinue supporting requests without parameters (which I find weird).